backend/tracim_backend/lib/core/user.py

```diff
@@ -243,9 +246,70 @@ def get_members_of_workspaces(self, workspace_ids: typing.List[int]) -> typing.L
         )
         return [item[0] for item in user_ids_in_workspaces_tuples]
 
+    def get_common_user_workspaces_for_user(self, user_id: int) -> List[Tuple[Any, ...]]:
+        """
+        Return list of UserRoleInWorkspace that are in common with the provided user
+        :param user_id: id of the user to get data from
+        :return: List of found UserRoleInWorkspace in common with the provided user
```

Comment:
if it returns a list of UserRoleInWorkspace, shouldn't the typing reflect that?

Reply:
Changed in last commit. Return type is the same but I changed the description

Reply:
fixed